src/durableorchestrationclient.ts

```diff
@@ -158,6 +158,7 @@ export class DurableOrchestrationClient {
                 },
             },
             maxContentLength: Infinity,
+            maxBodyLength: 5000000,

```

Comment:
nit: please comment this section
``suggestion
            maxBodyLength: 5000000, // 5MB

```